docs/user_guides/fs/feature_view/query.md

```diff
@@ -114,48 +192,48 @@ For the Scala part of the API, equivalent methods are available in the `Feature`
 
 === "Python"
     ```python
-    filtered_rain = rain_fg.filter(rain_fg.location_id == 10)
+    filtered_credit_card_transactions = credit_card_transactions_fg.filter(credit_card_transactions_fg.category == "Grocery")
     ```
 
 === "Scala"
     ```scala
-    val filteredRain = rainFg.filter(rainFg.getFeature("location_id").eq(10))
+    val filteredCreditCardTransactions = creditCardTransactionsFg.filter(creditCardTransactionsFg.getFeature("category").eq("Grocery"))
     ```
 
 Filters are fully compatible with joins:
 
 === "Python"
     ```python
-    feature_join = rain_fg.select_all() \
-        .join(temperature_fg.select_all(), on=["date", "location_id"]) \
-        .join(location_fg.select_all(), left_on=["location_id"], right_on=["id"], join_type="left") \
-        .filter((rain_fg.location_id == 10) | (rain_fg.location_id == 20))
+    selected_features = credit_card_transactions_fg.select_all() \
+        .join(account_details_fg.select_all(), on=["cc_num"]) \
```

Comment:
cc_num -> merchant_id

Reply:
its `account_details_fg`, cc_num is the foreign key here in `credit_card_transactions_fg` and `id` or `transaction_id` will be pk